app/Jobs/FixPolygonOverlapJob.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Http\Middleware\SetAuthenticatedUserForJob;
+use App\Models\DelayedJob;
+use App\Services\PolygonService;
+use Exception;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Http\Response;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Str;
+use Throwable;
+
+class FixPolygonOverlapJob implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+
+    private const STATUS_PENDING = 'pending';
+    private const STATUS_FAILED = 'failed';
+    private const STATUS_SUCCEEDED = 'succeeded';
+
+    protected $polygonService;
+
+    protected $polygonUuids;
+
+    protected $job_uuid;
+
+    public $authUserId;
+
+    /**
+     * Create a new job instance.
+     *
+     * @param array $polygonUuids
+     */
+    public function __construct(array $polygonUuids, int $authUserId)
+    {
+        $this->polygonUuids = $polygonUuids;
+        $this->job_uuid = Str::uuid()->toString();
+        $this->authUserId = $authUserId;
+
+    }
+
+    /**
+     * Get the middleware the job should pass through.
+     *
+     * @return array
+     */
+    public function middleware()
+    {
+        return [new SetAuthenticatedUserForJob];
+    }
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+
+        try {
+            DelayedJob::create([
+              'uuid' => $this->job_uuid,
+              'status' => self::STATUS_PENDING,
+              'created_at' => now(),
+            ]);
+            $user = Auth::user();
+            if ($user) {
+                $polygonsClipped = App::make(PolygonService::class)->processClippedPolygons($this->polygonUuids);
+                DelayedJob::where('uuid', $this->job_uuid)->update([
+                  'status' => self::STATUS_SUCCEEDED,
+                  'payload' => json_encode(['updated_polygons' => $polygonsClipped]),
+                  'updated_at' => now(),
```

Comment:
We should never be managing these timestamps ourselves. The HasTimestamps trait that's included in the base Model class does this for us.

Reply:
done

Reply:
Hmm, it looks like the `updated_at' => now()` line is still there in this instance.